krun/results.py

```diff
@@ -117,6 +122,9 @@ def integrity_check(self):
             if mperf_len != wct_len:
                 fatal("inconsistent mperf length: %s: %d vs %d" % (key, mperf_len, wct_len))
 
+            if pexec_flags_len != wct_len:
+                fatal("inconsistent mperf length: %s: %d vs %d" % (key, mperf_len, wct_len))
```

Comment:
I think the error message is wrong (it's not mperf length, right?).

Reply:
fixed in a later commit.

Reply:
Yes, I realised that later.